apps/web/components/molecules/userAvatar/userAvatar.tsx

```diff
@@ -0,0 +1,23 @@
+import Image from 'next/image';
+import clsx from 'clsx';
+
+type CommentUserAvatarProps = {
+	avatar: string;
+	size?: number;
+};
+
+export const UserAvatar = ({ avatar, size = 24 }: CommentUserAvatarProps) => {
+	const sizeClass = `w-${size} h-${size}`;
+	return (
+		<Image
+			alt=""
+			src={avatar}
```

Comment:
Is there a case where user doesn't have an avatar? Some form of fallback/default?

Reply:
I think that at the moment the only place where the avatar may be undefined with a header, but then the default avatar is passed in props

Reply:
Yeah. I'm not sure if it would be good to use that as a default inside the `Avatar` component. But as for now it's good enough